src/modules/rlm_eap/libeap/mppe_keys.c

```diff
@@ -219,18 +232,21 @@ void eapttls_gen_challenge(SSL *s, uint8_t *buffer, size_t size)
 	memcpy(p, s->s3->client_random, SSL3_RANDOM_SIZE);
 	p += SSL3_RANDOM_SIZE;
 	memcpy(p, s->s3->server_random, SSL3_RANDOM_SIZE);
+	p += SSL3_RANDOM_SIZE;
```

Comment:
I don't think this line is necessary.  Compilers may complain about unused assignments

Reply:
Resolved: https://github.com/FreeRADIUS/freeradius-server/pull/3516/commits/5deea57dd182cf575ef1d4c3a0288428aaed0ff4